src/Models/Interfaces.ts

```diff
@@ -0,0 +1,62 @@
+export interface JSONInterfacePhotosBySolRequest {

```

Comment:
usually each interface lives in its own file

you don't need `JsonInterface` in the interface name, do you?